test/run-stp-tests.sh

```diff
@@ -1025,7 +1029,15 @@ else
 fi
 
 # Error log messages
-count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | \
+       $GREP -v '(usb) crashed on signal 11' | \
+       $GREP -v '(dnssd) stopped with status 1' | \
+       $GREP -v 'loadFile failed: temp file: not a PDF file' | \
+       $GREP -v 'Failed to connect to system bus' | \
+       $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | \
```

Comment:
Ditto

Reply:
This one appears on IPv6-deprived hosts.